it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationControllerISpec.scala

```diff
@@ -219,52 +209,34 @@ trait BusinessIdentificationControllerISpec extends BaseISpec with SessionDataMi
       metricShouldExistAndBeUpdated("Count-Subscription-AlreadySubscribed-HasEnrolment-AgentServicesAccount")
     }
 
-    BusinessIdentificationForms.validBusinessTypes.foreach { validBusinessTypeIdentifier =>
+    validBusinessTypes.foreach { validBusinessTypeIdentifier =>
       s"show check Agency Status page for valid businessTypeIdentifier: $validBusinessTypeIdentifier" in {
         val result = await(
-          controller.showBusinessDetailsForm(Some(validBusinessTypeIdentifier))(
+          controller.showBusinessDetailsForm(validBusinessTypeIdentifier)(
             authenticatedAs(subscribingCleanAgentWithoutEnrolments)))
         status(result) shouldBe 200
-        val bodyString = bodyOf(result)
-        bodyString should include(htmlEscapedMessage("businessDetails.title"))
-        bodyString should include(htmlEscapedMessage(s"businessDetails.label.utr.$validBusinessTypeIdentifier"))
-      }
-    }
 
-    s"redirect to obtain identifier again, when Invalid businessTypeIdentifier" in {
```

Comment:
May still need this, but it looks very similar to another removed test on line 195 ("invalid businessType parameter supplied" )

Reply:
seems like the same test as per your comment above